xmtp/src/lib.rs

```diff
@@ -24,4 +29,15 @@ mod tests {
             Some(b"bar".to_vec())
         );
     }
+
+    #[test]
+    fn test_can_generate_test_contact_bundle_and_session() {
+        let bundle = generate_test_contact_bundle();
+        assert!(bundle.identity_key.is_some());
+        assert!(bundle.prekey.is_some());
+
+        // Generate an outbound session (Olm Prekey Message) given a VmacContactBundle
+        let session = generate_outbound_session(bundle);
+        assert!(!session.is_empty());
+    }
 }
```

Comment:
These tests could be placed in the same file as the functioned being tested. 

E.g. https://github.com/xmtp/libxmtp/blob/main/xmtp/src/builder.rs#L44

Reply:
did it!